process/elasticproc/block/blockProcessor.go

```diff
@@ -288,3 +327,26 @@ func createShardIdentifier(shardID uint32) uint32 {
 func (bp *blockProcessor) ComputeHeaderHash(header coreData.HeaderHandler) ([]byte, error) {
 	return core.CalculateHash(bp.marshalizer, bp.hasher, header)
 }
+
+func getTxsMap(mbType nodeBlock.Type, pool *outport.Pool) map[string]coreData.TransactionHandlerWithGasUsedAndFee {
+	switch mbType {
+	case nodeBlock.TxBlock:
+		return pool.Txs
+	case nodeBlock.InvalidBlock:
+		return pool.Invalid
+	case nodeBlock.RewardsBlock:
+		return pool.Rewards
+	case nodeBlock.SmartContractResultBlock:
+		return pool.Scrs
+	default:
+		return map[string]coreData.TransactionHandlerWithGasUsedAndFee{}
```

Comment:
return `make(map[string]coreData.TransactionHandlerWithGasUsedAndFee{})`, otherwise it will panic in L262 ?

Reply:
done.